src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Hl7v2/ADT_A01/ADT01-23-expected.json

```diff
@@ -192,51 +192,33 @@
       },
       "request": {
         "method": "PUT",
-        "url": "Location/5f49ca69-cbda-3b30-8480-fa750dfee4a2"
+        "url": "Location/9b2956e2-230b-9dd5-ad8d-0e697a8664c1"
       }
     },
     {
-      "fullUrl": "urn:uuid:a447a61d-cde0-394f-ab88-6a6f01215075",
+      "fullUrl": "urn:uuid:b8e7e176-84c6-7e2d-d422-a69df56229d8",
       "resource": {
         "resourceType": "RelatedPerson",
-        "id": "a447a61d-cde0-394f-ab88-6a6f01215075",
-        "address": [
```

Comment:
We should be able to generate address and name of RelatedPerson. This should not be removed from groundtruth.

Reply:
Solved by converting back `Repeats[0]` and udpating `Resource/Immunization`.